notebooks/tutorials/landscape_evolution/erosion_deposition/erosion_deposition_component.ipynb

```diff
@@ -0,0 +1,266 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"http://landlab.github.io\"><img style=\"float: left\" src=\"../../../landlab_header.png\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to the `ErosionDeposition` component"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<hr>\n",
+    "<small>For more Landlab tutorials, click here: <a href=\"https://landlab.readthedocs.io/en/latest/user_guide/tutorials.html\">https://landlab.readthedocs.io/en/latest/user_guide/tutorials.html</a></small>\n",
+    "<hr>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial introduces the `ErosionDeposition` component, which simulates erosion and deposition in a river network over long time scales.\n",
+    "\n",
+    "## Theory and Overview\n",
+    "\n",
+    "`ErosionDeposition` models fluvial processes using the approach described by Davy and Lague (2009). The basic goal is to calculate rates of erosion or deposition in the various branches of a river network that is embedded in a gridded landscape (see, e.g., Tucker and Hancock, 2010). The literature has a number of different approaches to this problem. Some models assume transport-limited conditions, such that erosion or deposition result from local imbalances in sediment transport capacity (see, e.g., Willgoose, 2018). Some assume detachment-limited conditions, such that any eroded sediment is entirely removed  (e.g., Howard, 1994; Whipple and Tucker, 1999; Tucker and Whipple, 2002). Still others allow for both erosion of detachment-resistant material, and re-deposition of that material. The Davy-Lague approach falls in the latter category, and is similar the approach used in some shorter-term morphodynamic models as well as agricultural soil erosion models. The basic idea involves conservation of sediment mass in the water column of a river. One calculates the rate of entrainment of sediment from the bed into the water as a function of discharge and local slope. The bed also accumulates sediment that settles out from the water column, at a rate that depends on sediment concentration (treated as the ratio of sediment flux to water discharge) and a settling-velocity parameter.\n",
```

Comment:
The sentence, "Still others allow for both erosion of detachment-resistant material, and re-deposition of that material" is ambiguous as to what is represented by "detachment-resistant material" - does that mean bedrock? Or very large mobile clasts that are difficult to entrain? If it's acceptable for the modeler to think of the "detachment-resistant material" as bedrock, it's probably best to call it that. Otherwise, maybe just include a couple of examples of a range of physical conditions that would be represented as "detachment-resistant material" in the model.

Reply:
added "(bedrock or cohesive sediment)"